backend/src/password/PasswordEncryptr.ts

```diff
@@ -0,0 +1,6 @@
+import { User } from '@entity/User'
+
+export interface PasswordEncryptr {
+  encryptPassword(dbUser: User, password: string): Promise<bigint>
+  verifyPassword(dbUser: User, password: string): Promise<boolean>
```

Comment:
i am not sure if js allows to implement an function in the interface and allows to override it in the class implementing this interface, but this function is the same in both encryptors.

If its possible you can also put the password check into the interface (has the password the right amount of character-/types)